rasa/shared/core/domain.py

```diff
@@ -1022,12 +1030,16 @@ def concatenate_entity_labels(
 
         if entity:
             return [
-                f"{entity}{rasa.shared.core.constants.ENTITY_LABEL_SEPARATOR}{sub_label}"
+                f"{entity}"
+                + f"{rasa.shared.core.constants.ENTITY_LABEL_SEPARATOR}"
```

Comment:
why do we need to concatenate via `+`?

Reply:
Honestly, I don't know kk
I'll take a look at that when reviewing the imports.